third_party/move/move-model/src/builder/exp_builder.rs

```diff
@@ -2445,6 +2448,7 @@ impl<'env, 'translator, 'module_translator> ExpTranslator<'env, 'translator, 'mo
         maccess: &EA::ModuleAccess,
         generics: &Option<Vec<EA::Type>>,
         fields: Option<&EA::Fields<EA::LValue>>,
+        no_args_check: bool, // skip evaluating any arguments
```

Comment:
The negated sense of this parameter is a bit confusing.  Can you negate the sense and call it `check_args`?

Reply:
Gone